code/API_definitions/device-status.yaml

```diff
@@ -1,7 +1,72 @@
 openapi: 3.0.3
 info:
   title: Device Status
-  description: Service Enabling Network Function API for monitoring device status. This API allows to query device roaming status and subscribe to event related to device roaming status.
+  description: |
+    This API provides the customer with the ability to query the roaming status of a device, and to subscribe to receive notifications when the device roaming status changes.
+
+    # Introduction
+
+    The API consumer is able to verify whether a certain user device is in a roaming situation (or not). This capability is provided in two ways:
+      - via a direct request, with the current roaming status given in the response.
+      - via a subscription request - in this case the roaming situation is not in the response but a notification is sent back to the subscriber when the device roaming status changes
+
+    The verification of the roaming situation depends on the network's ability. In addition to the roaming status, visited country information is included in the response when the device is roaming.
+
+    Device roaming status verification could be useful in scenarios such as (not exhaustive):
+      - For regulatory reasons, where a customer may need to be within a certain jurisdiction, or outwith others, in order for transactions to be authorized
+      - For security / fraud / fair play reasons, to establish that a customer is located where they claim to be
+      - For service delivery reasons, to ensure that the customer has access to particular service, and will not incur roaming charges in accessing them
+
+
+    # Relevant terms and definitions
+
+    * **Device**: A device (i.e. user equipment) refers to any physical entity that can connect to a mobile network and participate in network communication.
+      At least one identifier for the device must be provided from the following four options:
+      - IPv4 address
+      - IPv6 address
+      - Phone number (MSISDN)
+      - Network Access Identifier assigned by the mobile network operator for the device
```

Comment:
Should we maybe add a note that when you receive **Device** in a response or notification you will not get more device identifiers back than you used in the request?

Reply:
I added a note to this effect